src/docker/compose.local.yaml

```diff
@@ -1,31 +1,48 @@
 services:
+  aptos-node:
+    network_mode: host
+    build:
+      context: ../../
+      dockerfile: src/docker/aptos-node/Dockerfile
+    ports:
+      - "50051:50051"
+      - "8070:8070"
+      - "8090:8090"
+      - "8080:8080"
+      - "8081:8081"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8070/"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+      start_period: 30s
+    command: ["bash", "/app/run-fullnode.sh"]
+    # In order to start the node with the indexer API; i.e., run
+    # `aptos node run-local-testnet --with-indexer-api`, we must
+    # mount the Docker socket to the container.
+    # This effectively allows the container to start a new container.
+    # Note that this is *not* "Docker-in-Docker", but rather
+    # "Docker-outside-Docker".
+    # That is, the container can act as the host in terms of creating
+    # and orchestrating other containers but still appears as a sibling
+    # to the existing containers and any new containers it creates.
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Is this known to work on Linux too?

Reply:
I would assume so since this was actually included in #210 originally, I just didn't realize it/know what it was doing. And I imagine Bogdan it all to work on his machine (this file originally only changed because I couldn't build the node with an install script on Mac, so I had to fully build from the repository)